docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+bot:
+  build: .
+  volumes:
+    - .:/web
+  ports:
+    - "8800:80"
+  command: /bin/bash -c "cd /web/;
```

Comment:
Easy to update dependencies without rebuild `docker-compose exec bot pip install -r requirements.txt`